StudyBuddy/Views/Chat/ChatWithUser.cshtml

```diff
@@ -122,6 +119,21 @@
     </div>
 </main>
 
+    <script>
+        window.onload = function () {
+            document.getElementById('messageInput').focus();
+        };
+
+        // Function to handle the key press event
+        document.getElementById('messageInput').addEventListener('keypress', function (event) {
```

Comment:
Add a check for empty messageInput field to prevent empty messages getting sent and taking up space on db. You can also look into adding a delay to prevent spam.